### PR TITLE
Fixed failed Unit tests in VerticalBarChart, VerticalStackedBarChart, GroupedVerticalBarChart and SankeyChart

### PR DESCRIPTION
**Pull request by v-baambati**
Fixed failed Unit tests in VerticalBarChart, VerticalStackedBarChart, GroupedVerticalBarChart and SankeyChart

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

✅ No changes found
<div id="bundleSizeComment9423"/>

File: change/@fluentui-react-charting-d3c8a165-c5d9-4b99-a8c6-f50ee0409345.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fabricteam) [inline change/@fluentui-react-charting-d3c8a165-c5d9-4b99-a8c6-f50ee0409345.json:1]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31376&commitId=1aed335551799f59bdfdd62319127827709558fb&env=prod&clientType=FLUENTUI)


  <details><summary><b>Pivot - Overflow</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Tabs - RTL.Narrow - Last tab selected.chromium.png|0| Removed|
|Pivot - Overflow.Tabs - RTL.Narrow - Overflow menu.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

**Review by AtishayMsft — APPROVED** [review]

File: change/@fluentui-react-charting-d3c8a165-c5d9-4b99-a8c6-f50ee0409345.json (hunk 1 above)
Comment (AtishayMsft) [reply change/@fluentui-react-charting-d3c8a165-c5d9-4b99-a8c6-f50ee0409345.json:1]: Unrelated change

**Pull request merged**